napari/layers/utils/layer_utils.py

```diff
@@ -958,3 +958,18 @@ def _features_to_properties(features: pd.DataFrame) -> Dict[str, np.ndarray]:
     :meth:`_FeatureTable.properties`
     """
     return {name: series.to_numpy() for name, series in features.items()}
+
+
+def _unique_element(array: Array):
+    """
+    Returns the unique element along the 0th axis, if it exists; otherwise, returns None.
+
+    This is faster than np.unique, does not require extra tricks for nD arrays, and
+    does not fail for non-sortable elements.
+    """
+    if len(array) == 0:
+        return None
+    el = array[0]
+    if np.any(array[1:] != el):
```

Comment:
Is this statement lazy? Or do we immediately materialize the full binary comparison? Doesn't really matter because I think performance is already pretty, but just thinking if there's a small improvement to make.

Reply:
What do you mean with lazy? It's not looping through and stopping early, if that's what you mean, but I doubt that would be faster than a vectorized numpy function.

Reply:
Well, I assume it would be faster in some cases to iterate (e.g. when array is long, but the unequal element is early), but my guess is on average it won't make much difference, and in some cases it might be a lot slower. So current code looks good to me.